app/assets/javascripts/components/resources/handouts.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Block from '../timeline/block';
+
+const handouts = [
+  { name: 'Editing Wikipedia', link: 'https://wikiedu.org/editingwikipedia' },
+  { name: 'Evaluating Wikipedia', link: 'https://wikiedu.org/evaluatingwikipedia' },
+  { name: 'Illustrating Wikipedia', link: 'https://wikiedu.org/illustratingwikipedia' }
+];
+
+const Handouts = ({ blocks, trainingLibrarySlug }) => {
+  let topicGuides;
+  const topicGuidesBlock = blocks.find(block => block.title.match(/writing articles in your topic area/));
+  if (topicGuidesBlock) {
+    topicGuides = (
+      <>
```

Comment:
What's happening here?

Reply:
This is syntax for React fragments that I learned about from an error message today: `<>...</>` :boom: